### ISSUE TITLE
Docs still reference fasttext.build_vocab sentences parameter

### ISSUE THREAD

**Issue by Jeff-Winchell** [piskvorky/gensim#3501]
Current version of Fasttext.build_vocab method has a first parameter with two possible names, neither of which is "sentences" despite that 2+ year-old (pre version 4) parameter name being listed on https://radimrehurek.com/gensim/models/fasttext.html

**Comment by gojomo** [comment]
Yep, see #3152. PRs that ensure the preferred, explicit 'corpus_iterable' & 'corpus_file' are used everywhere (while the 'sentences' doesn't break where it wa working in any 4.x) – & docs match, showing 'sentences' as deprecated – would make sense to me.
